.drone.yml

```diff
@@ -12,11 +13,48 @@ steps:
       recursive: true
       tags: true
   - name: build
-    image: plugins/docker
+    image: drgrove/drone-kaniko@sha256:e3045421c3683e6baf5628b22ea0ee1cd7ae217f4de0e1bc53a0a1a20335b108
```

Comment:
why your fork?
Please at least link the commit this image is from in a comment

Reply:
The archived the repo and haven't updated it in 16 months